frontend/src/components/AnomalyTable.vue

```diff
@@ -86,18 +86,18 @@
   <BBModal
     v-if="state.showModal"
     :title="`'${state.selectedAnomaly.database.name}' schema drift - ${state.selectedAnomaly.payload.version} vs Actual`"
-    @close="dimissModal"
+    @close="dismissModal"
   >
     <div class="space-y-4">
       <code-diff
         class="w-full"
-        :old-string="state.selectedAnomaly.payload.expect"
-        :new-string="state.selectedAnomaly.payload.actual"
+        :oldString="state.selectedAnomaly.payload.expect"
+        :newString="state.selectedAnomaly.payload.actual"
         :fileName="`${state.selectedAnomaly.payload.version} (left) vs Actual (right)`"
-        output-format="side-by-side"
+        outputFormat="side-by-side"
```

Comment:
Because we widely use var names like `selectedID` in backend code (which Golang prefer). Using hyphenated attrs will make it `selected-i-d`. It looks really weird. 
I prefer to keep `selectedID` or `outputFormat` camelized to make all stack the same. Even though this doesn't follow vue/recommended style.

Reply:
updated, re-enabled `vue/attribute-hyphenation`. 
@xiaoluoboding PTAL